### PR TITLE
Removing hover opacity change for pod-donut defocused states and …

### PR DESCRIPTION
**Pull request by sg00dwin**
…turning on expand behavior. And center tooltip correctly

Fixes https://github.com/openshift/origin/issues/9253

pod-donut

### CODE REVIEW

File: app/scripts/directives/podDonut.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ angular.module('openshiftConsole')
           type: "donut",
           bindto: '#' + $scope.chartId,
           donut: {
-            expand: false,
+            expand: true,
```
Comment (spadgett) [inline app/scripts/directives/podDonut.js]: This was disabled when we added empty donuts when there are no pods. Can you double check this case?

https://github.com/openshift/origin/pull/5464#discussion-diff-43297113

Comment (sg00dwin) [inline app/scripts/directives/podDonut.js]: yeah, I assumed we wanted to switch behaviors (remove opacity change and add expand). Maybe we can disable the expand for empty donuts?

**Comment by sg00dwin** [comment]
@spadgett reverted expand to false

pod-donut

**Comment by spadgett** [comment]
@sg00dwin Yeah I don't see a way in the API to toggle `expand` after the chart is created for empty charts. It's why we just disabled it entirely.

@jwforres Thoughts on this change?

**Comment by jwforres** [comment]
I do prefer it without the opacity change. But it feels more interactive with the expansion.  How weird does it actually look for the empty donut to expand?

**Comment by spadgett** [comment]
> How weird does it actually look for the empty donut to expand?

It's not awful, but there's no tooltip so a little weird. I just pulses on mouse over.

File: app/styles/_components.less [hunk 2]
```diff
@@ -281,6 +281,19 @@ code.probe-command {
...
+    opacity: 1 !important;
+  }
+  .c3-tooltip-container {
+    // centered above donut
+    left: 50% !important;
+    top: -10px !important;
+    transform: translate(-50%, -50%);
+    // prevent text wrap
+    white-space: nowrap;
+  }
```
Comment (spadgett) [inline app/styles/_components.less]: Can you remove the following lines with this change (and the trailing comma on the line above)?

https://github.com/spadgett/origin-web-console/blob/339a32eff93ba3a1298717fe2e9f82f55ea0d884/app/scripts/directives/podDonut.js#L84-L87

**Comment by sg00dwin** [comment]
urzbgo1sz6

File: app/styles/_components.less (hunk 2 above)
Comment (spadgett) [inline app/styles/_components.less]: Or maybe we can set just `white-space: nowarp` and let c3 position the tooltip.

**Comment by spadgett** [comment]
We could add a custom tooltip when there are no pods. Or disable the expand in CSS for empty styles (not sure if possible).

**Comment by jwforres** [comment]
Yeah i considered a custom tooltip as well. Something simple like "No pods
exist".   If disabling the expand from CSS for the empty state is possible,
I would prefer that option.

**Comment by sg00dwin** [comment]
I looked closer at the expand to see if there's a way to disable it, using css, for empty pods and found that it's the actual `svg` path coordinates that are being transformed. So css wouldn't effect it.

@spadgett updated pr

**Comment by sg00dwin** [comment]
@ajacobs21e 
Here's the [screenshot gif](https://cloud.githubusercontent.com/assets/1874151/20571334/fd5b1d44-b174-11e6-8e2f-4cdc3ddc505e.gif) showing the hover expand over empty pod issue.

**Comment by ajacobs21e** [comment]
@sg00dwin are you able to add in a tooltip for an empty donut?

**Comment by ajacobs21e** [comment]
Custom tooltip with "No Pods" or "Zero Pods" ? @spadgett

**Comment by sg00dwin** [comment]
@spadgett, @jwforres 
Updated the pr with 'No pods exist' tooltip for empty state.

no-pods-tooltip

**Comment by spadgett** [comment]
[merge]

**Pull request merged**